configs/workers-cloudflare-v2.json

```
@@ -0,0 +1,15 @@
+{
+  "index_name": "workers-cloudflare-v2",
+  "start_urls": ["https://workers.cloudflaredocs.workers.dev"],
+  "stop_urls": [],
+  "sitemap_urls": ["https://workers.cloudflaredocs.workers.dev/sitemap.xml"],
+  "selectors": {
+    "lvl0": ".DocsContent h1",
+    "lvl1": ".DocsContent h2",
+    "lvl2": ".DocsContent h3",
+    "lvl3": ".DocsContent h4",
+    "lvl4": ".DocsContent h5",
+    "lvl5": ".DocsContent h6",
+    "text": ".DocsContent p, .DocsContent li"
+  }
```

```suggestion
  },
    "conversation_id": [
    "1253962701"
  ],
```

We will use the conversation create by @davidtsong in the meantime

---

got it! adding that in my own commit to do a little indentation fix